kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/DocumentServiceBase.java

```diff
@@ -117,13 +117,13 @@ public DocumentInstanceList listDocuments(Integer page, Integer pageSize) {
 
         List<DocumentInstance> list = new ArrayList<DocumentInstance>();
         for (Document doc : documents) {
-            list.add(convertDocument(doc));
+            list.add(convertDocument(doc, false));
         }
 
         return list;
     }
 
-    protected DocumentInstance convertDocument(Document document) {
+    protected DocumentInstance convertDocument(Document document, boolean withContent) {
```

Comment:
Tbh, I am not a big fan of these boolean flagging args :) I would prefer enum (even with just two values). But I won't hold the PR for this, since it is largely matter of personal style.